<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrackingRecording</class>
 <widget class="QDialog" name="TrackingRecording">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据记录</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="m_timeLabel">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>00:00:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_statusLabel">
     <property name="text">
      <string>状态: 准备就绪</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_startBtn">
       <property name="text">
        <string>开始记录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_stopBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>停止记录</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>

